02-Reproject-Raster-In-R.Rmd

```diff
@@ -42,6 +43,12 @@ After completing this activity, you will:
 
 * Be able to reproject a raster in R
 
+###Challenge Code
```

Comment:
Ok - let's talk more about the grey box content. i see both sides. it's getting long is my concern and search engines look for content towards the top first. So it's good to have key content at the top. if all our pages "look" the same it detracts. But also it just gets visually long. let's brainstorm!